src/main/java/org/opentripplanner/graph_builder/module/osm/DefaultWayPropertySetSource.java

```diff
@@ -61,7 +61,13 @@
     @Override
     public WayPropertySet getWayPropertySet() {
         WayPropertySet props = new WayPropertySet();
-
+        populateProperties(props);
+        return props;
+        
+    }
+    
+    /* Populate properties on existing WayPropertySet. Makes it easer to override any properties by sub classes. */
+    protected void populateProperties(WayPropertySet props) {
```

Comment:
I see that you're making these protected so they are accessible from a subclass, but I'm now realizing maybe they should be `public static` utility functions on the superclass. They don't seem to need any state fields from the class instance itself.